partition/bootloader.go

```diff
@@ -50,10 +49,10 @@ var (
 // bootloader
 type Bootloader interface {
 	// Return the value of the specified bootloader variable
-	GetBootVar(name string) (string, error)
+	GetBootVars(names []string) (map[string]string, error)
```

Comment:
Wouldn't making it `GetBootVars(names ...string)` make all the uses of `GetBootVars` nicer?